modules/ocf_desktop/files/xsession/lab_close_notify

```diff
@@ -0,0 +1,37 @@
+#!/usr/bin/env python3
+
+import datetime
+import sched
+import subprocess
+import time
+
+from ocflib.lab import hours
+
+# 0 minutes, 2 minute, 5 minutes, 10 minutes, 15 minutes
+WARNING_DURATIONS = [0, 120, 300, 600, 900]
+
+def notify_user(seconds_left):
+    title = 'Lab closure'
+    if seconds_left != 0:
+        msg = 'Lab closing in {0:g} minutes'.format(seconds_left / 60)
+    else:
+        msg = 'Lab is closing NOW!'
+
+    # TODO: pick an icon
+    subprocess.call(['notify-send', title, msg])
+
+today = datetime.datetime.today()
```

Comment:
I get the reasoning for this, but it seems a bit overkill for a small script that I certainly don't expect anyone to import. However if this is really the style for all Python code at the OCF then I wont push it.